aleph-client/src/pallets/aleph.rs

```diff
@@ -81,6 +83,14 @@ impl<C: ConnectionApi> AlephApi for C {
 
         self.rpc_call(method.to_string(), params).await.unwrap()
     }
+
+    async fn emergency_finalizer(&self, at: Option<BlockHash>) -> Option<AccountId> {
+        let addrs = api::storage().aleph().emergency_finalizer();
+
+        self.get_storage_entry_maybe(&addrs, at)
+            .await
+            .map(|public| public.0 .0.into())
```

Comment:
Why do we map to AccountId? This is just a "raw" ed25519 pubkey, it is not associated with any particular account

Reply:
I see you use raw bytes in your script. Would it be ok here to return [u8; 32]?

Reply:
Yes. I think it's fine, because you normally query it to check against some other key (in whatever format).

Reply:
done